cmd/skopeo/signing.go

```diff
@@ -88,3 +89,137 @@ var standaloneVerifyCmd = cli.Command{
 	ArgsUsage: "MANIFEST DOCKER-REFERENCE KEY-FINGERPRINT SIGNATURE",
 	Action:    standaloneVerify,
 }
+
+func addSignature(context *cli.Context) error {
+	if len(context.Args()) != 2 {
+		return errors.New("Usage: skopeo add-signature reference key-fingerprint")
+	}
+	ref := context.Args()[0]
+	fingerprint := context.Args()[1]
+
+	imageRef, err := transports.ParseImageName(ref)
+	if err != nil {
+		return fmt.Errorf("Error parsing %s: %v", ref, err)
+	}
+	name := transports.ImageName(imageRef)
```

Comment:
The value we get back can be different from the passed-in value (IIRC the docker reference will include the default `latest` tag if the incoming value doesn't contain one, and having a signature check fail because either the signed or expected value included that tag and the other one didn't got annoying pretty fast.